packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -909,7 +928,7 @@ impl<'m> RunnerImpl<'m> {
             payload_str.clone(),
         ]);
 
-        let (next_task_msg, warnings, logs) = self.run_task(
+        match self.run_task(
```

Comment:
Can we write to a temporary variable (e.g. `run_result`) and `match run_result`? IMO this is more readable.

Reply:
Done